content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -182,7 +182,7 @@ different Kubernetes components.
 | `ServerSideFieldValidation` | `false` | Alpha | 1.23 | - |
 | `ServiceInternalTrafficPolicy` | `false` | Alpha | 1.21 | 1.21 |
 | `ServiceInternalTrafficPolicy` | `true` | Beta | 1.22 | |
-| `ServiceIPStaticSubrange` | `false` | Alpha | 1.24 | |
+| `ServiceIPStaticSubrange` | `true` | Beta | 1.25 | |
```

Comment:
Please refer to `ServiceInternalTrafficPolicy` to see how we document BOTH 'Alpha' and 'Beta' stages.
Note we are not simply replacing the 'Alpha' line.

Reply:
I believe the `content/en/docs/concepts/services-networking/service.md` file needs an update as well.

Reply:
yeah, thanks